docs/pages/contributing/documentation/terms-usage.mdx

```diff
@@ -0,0 +1,171 @@
+## Terminology and usage conventions
+
+This section covers common terminology, style, and usage questions.
+It includes recommended practices for using punctuation, referring to user interface
+elements, and selecting the right words for the information you're trying to convey. 
+
+### Above and below
+
+Don't use *above* to mean *earlier* or as an adjective preceding a noun (*the above section*) 
+or following a noun (*the code above*). Use a hyperlink, or use *previous*, *preceding*, or 
+*earlier*. Don't use *below* to mean *later* or as an adjective preceding a noun (*the below 
+section*) or following a noun (*the code below*). Use a hyperlink, or use *later* or 
+*the following*. For example: 
+
+- Use the preceding code to display information about the database.
+- Use the following code to display information about the database.
+
+### Dates and numbers
+
+Date formats can depend on context, including the location of a computer and a user's 
+preferences. In most cases, you can use the DD Mon YYYY or DD Month YYYY format for dates. 
+In body text, spell out whole numbers from zero through nine. Use numerals for 10 or greater. 
+Use commas in numbers with four or more digits. Use *more than* instead of *over* 
+(over is a spatial term). 
+
+### Emphasis and special notices
+
+Use bold formatting for user interface elements that the user interacts with, including:
+
+- Menus and menu commands.
+- Field and button labels.
+- Options that can be selected in the user interface.
+
+If there's text that requires more attention than the surrounding body, consider isolating 
+it as a standalone note or tip. However, **think twice** before adding any type of special 
+notice, embedding details, or using tabbed boxes. These elements can all be distracting and 
+too many of them make text unreadable. 
+
+Because they are generally **disruptive** to the reader's experience, ask yourself if it is 
+really necessary to stop the reader's forward progress by adding a Note, Caution, Tip, or 
+any other special component. In most cases, it's more effective to integrate the information 
+in the main body of the text. If you use these types of components at all, use them sparingly! 
+
+**Note**—Indicates neutral or positive information that emphasizes or supplements important 
+points of the main text. A note supplies information that may apply only in special cases. 
+Examples are memory limitations, equipment configurations, or details that apply to specific 
+versions of a program.
+
+**Tip**—Helps users apply the techniques and procedures described in the text to their specific 
+needs. A tip suggests alternative methods that may not be obvious and helps users understand 
+the benefits and capabilities of the product. A tip is not essential to the basic understanding 
+of the text.
+
+**Caution**—Advises users that failure to take or avoid a specific action could result in 
+loss of data.
```

Comment:
I'm not sure if this glossary is necessary. These aren't components in Teleport's component library, and we don't use these terms outside of this seciton.